apps/designer/package.json

```diff
@@ -43,7 +43,7 @@
     "@radix-ui/react-tabs": "^0.1.5",
     "@radix-ui/react-toggle": "^0.1.4",
     "@radix-ui/react-toggle-group": "^0.1.5",
-    "@radix-ui/react-tooltip": "^0.1.7",
+    "@radix-ui/react-tooltip": "^1.0.0",
```

Comment:
we should probably try to upgrade them all unless it breaks something) also feel free to separate the upgrade

Reply:
yeah, should definitely be another PR

Reply:
tried it and unfortunatly we need to see why it all errors out

![image](https://user-images.githubusercontent.com/1051509/181220670-7c5bbda2-fdc4-4a7d-81ec-821d754fd6b6.png)